Resources/views/tagmanager_body.html.twig

```diff
@@ -1,4 +1,4 @@
 <!-- Google Tag Manager (noscript) -->
-<noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ id }}"
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ id }}{{ url_suffix }}"
```

Comment:
@chrs-myrs @xyNNN I wonder if this is going to work without |raw filter (& is escaped and rendered as &amp;) Can you please investigate? I think it is not usable right now.